internal/services/containers/kubernetes_cluster_resource.go

```diff
@@ -1707,6 +1719,7 @@ func resourceKubernetesClusterRead(d *pluginsdk.ResourceData, meta interface{})
 			d.Set("private_link_enabled", accessProfile.EnablePrivateCluster)
 			d.Set("private_cluster_enabled", accessProfile.EnablePrivateCluster)
 			d.Set("private_cluster_public_fqdn_enabled", accessProfile.EnablePrivateClusterPublicFQDN)
+			d.Set("run_command_enabled", !utils.BoolDefault(accessProfile.DisableRunCommand, false))
```

Comment:
I appreciate the effort to be DRY here with the addition of the `BoolDefault` function - we could look into adopting this across the whole provider in the future since there are _a lot_ of these checks, but for the moment it's inconsistent. Would you please inline this check as was initially suggested?